api/src/main/java/com/capitalone/dashboard/webhook/github/GitHubV3.java

```diff
@@ -0,0 +1,201 @@
+package com.capitalone.dashboard.webhook.github;
+
+import com.capitalone.dashboard.ApiSettings;
+import com.capitalone.dashboard.client.RestClient;
+import com.capitalone.dashboard.custom.GitHubParsed;
+import com.capitalone.dashboard.misc.HygieiaException;
+import com.capitalone.dashboard.model.Collector;
+import com.capitalone.dashboard.model.CollectorItem;
+import com.capitalone.dashboard.model.CollectorType;
+import com.capitalone.dashboard.model.QCollectorItem;
+import com.capitalone.dashboard.model.webhook.github.GitHubRepo;
+import com.capitalone.dashboard.service.CollectorService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.mysema.query.BooleanBuilder;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.bson.types.ObjectId;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.ParseException;
+import org.springframework.data.querydsl.QueryDslPredicateExecutor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestClientException;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.HashMap;
+import java.util.Map;
+
+public abstract class GitHubV3 {
+    private static final Log LOG = LogFactory.getLog(GitHubV3.class);
+
+    private static final String REPO_URL = "url";
+    private static final String BRANCH = "branch";
+    protected static final String USER_ID = "userID";
```

Comment:
any reason why these need to be protected? they can be marked private

Reply:
This was by oversight, have made them private, thanks